backend/layers/processing/process_download.py

```diff
@@ -16,12 +22,22 @@
 from backend.layers.processing.logger import logit
 from backend.layers.processing.process_logic import ProcessingLogic
 from backend.layers.thirdparty.s3_provider_interface import S3ProviderInterface
+from backend.layers.thirdparty.step_function_provider import StepFunctionProvider
 from backend.layers.thirdparty.uri_provider import UriProviderInterface
 
+MEMORY_MODIFIER = 1.1  # add 10% overhead
+MEMORY_PER_VCPU = 4000
+MIN_MEMORY_MB = 4000
+MIN_VCPU = 1
```

Comment:
I think it's good to start with 1, but there is a chance that only using a single CPU, for a compute intensive process, may cause issues (e.g. send the machine into a resource starved state). Recommend revisiting this only if we see that processing of small datasets takes too long.